manifests/master/pdf_stack.pp

```diff
@@ -28,13 +28,7 @@
     require             => Yumrepo['robert-gcj'],
   }
 
-  # Regression in 0.12.4, https://github.com/wkhtmltopdf/wkhtmltopdf/issues/3241
```

Comment:
yes, that kept throwing an error so I left it in.  Should I pull it into another PR @binford2k